<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pySlicer</class>
 <widget class="qMRMLWidget" name="pySlicer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>295</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="L_Version">
     <property name="text">
      <string>pySlicer v1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="L_scriptpath">
     <property name="text">
      <string>path to the python script:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ctkPathLineEdit" name="PLE_ScriptPath"/>
   </item>
   <item>
    <widget class="QPushButton" name="pB_Load">
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="pTE_TextEditor"/>
   </item>
   <item>
    <widget class="QPushButton" name="pB_Run">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
